libstuff/libstuff.cpp

```diff
@@ -1318,10 +1319,9 @@ string SToJSON(const string& value, const bool forceString) {
     }
 
     // Otherwise, it's a string -- escape and return
-    // We need to escape all control characters in the string, not just the
-    // white-space control characters.
-    return "\"" + SEscape(value, "\x01\x02\x03\x04\x05\x06\x07\b\t\n\x0b\f\r\x0e\x0f\x10\x11"
```

Comment:
It looks like you removed the `\b`, `\t`, `\n`, `\f` and `\r` values. I'm not sure what they are...but is it really ok to get rid of them and stop escaping them?